checkstyle-tester/my_check.xml

```diff
@@ -27,10 +27,15 @@
          <!-- Example of sevntu.checkstyle Check usage -->
          <!-- <module name="NestedSwitchCheck"/> -->
 
-         <!-- usuppress javadoc parsing errors, as we test Check not a parser -->
+         <!-- suppress javadoc parsing errors, as we test Check not a parser -->
          <module name="SuppressionXpathSingleFilter">
             <property name="message" value="Javadoc comment at column \d+ has parse error"/>
          </module>
+
+         <!-- as we run on regression even on non-compiled files we need to skip exceptions on them -->
+         <property name="skipFileOnJavaParseException" value="true"/>
```

Comment:
Please put them under other Checker properties. Group them together above , before first inner module

Reply:
> Please put them under other Checker properties.

This is not checker property. Do you mean at the first line in tree walker module? like this

```XML
    <module name="TreeWalker">
         <!-- as we run on regression even on non-compiled files we need to skip exceptions on them -->
         <property name="skipFileOnJavaParseException" value="true"/>
         <property name="javaParseExceptionSeverity" value="ignore"/>

         <!-- Example of checkstyle Check usage -->
         <!-- PLEASE CHANGE IT TO CHECK YOU ARE TESTING !!!! -->
         <module name="ThrowsCount">
             <property name="max" value="20000000"/>
         </module>

         <!-- Example of sevntu.checkstyle Check usage -->
         <!-- <module name="NestedSwitchCheck"/> -->

         <!-- suppress javadoc parsing errors, as we test Check not a parser -->
         <module name="SuppressionXpathSingleFilter">
            <property name="message" value="Javadoc comment at column \d+ has parse error"/>
         </module>
    </module>
```

Reply:
I prefer to put it after the filter and leave the check modules above, right after `<module name="TreeWalker">`

Reply:
Treewalker, yes, all propertties are on most top before any inner modules.

Reply:
Okay, done